python/paddle/tensor/search.py

```diff
@@ -17,8 +17,8 @@
 from ..fluid.layer_helper import LayerHelper
 from ..fluid.data_feeder import check_variable_and_dtype, check_type, check_dtype
 from ..fluid import layers
```

Comment:
layers.* is replaced with other API, delete this line?

Reply:
OK